src/feature/followUnfollow/__tests__/followUnfollowAction.test.js

```diff
@@ -0,0 +1,190 @@
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+import axios from 'axios';
+import {
+  getFollowing, followAuthor, unfollowAuthor, clearFollowing
+} from '../followUnfollowAction';
+import LocalStorage from '../../../__mocks__/localStorage';
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+const store = mockStore({});
+
+let storage;
+const token = 'Invalid token';
+describe('Follow Author Tests', () => {
```

Comment:
@raymond42 the first argument in the describe block should be the name of the component under test., I think it should be 'FollowUnfollow ActionCreator'.